### ISSUE TITLE
vminventory --list should be --ansible?

### ISSUE THREAD

**Issue by pgrange** [n0vember/nocloud#11]
vminventory returns the list of running vms. The option --list displays it in json format for, it seems, ansible. Shouldn't this option be called --ansible?

vminventory --list
{
  "lvm_tst" : {
    "hosts" : [  "192.168.1.124", ],
  },
  "lvm" : {
    "children" : [ "lvm_tst", ],
    "vars": {
      "ansible_ssh_common_args": "-o StrictHostKeyChecking=no",
      "ansible_user": "root",
    },
  },
}

**Comment by n0vember** [comment]
No no :)

Ansible can use dynamic inventory. Constraints are :
- File is executable
- If given --list it returns the inventory
- If given --host <hostname> it returns variables for given host
- output is a defined json format

So no, I can't change the name of the --list option.

What I can do, is to return the same output as --list if given --ansible. But does it add any value ?

The output of vminventory without argument is just a use of the fact that nothing is defined for what the script should do if not given any argument. Probably the goal is to send an help message. I use this to give a basic list of running vm. In older nocloud versions there was a specific utility vmrunning for that.

vminventory also does not respond to --group. It returns nothing and so does not allow to give hosts variables this way. I tend to use group_vars for this, so it isn't really a necessity for me.

So basically, vminventory goal is to be called like this : 
```bash
ln -s /usr/local/bin/vminventory inventory
ansible-playbook -i inventory playbook.yml
```

[ansible doc on developing dynamic inventories](http://docs.ansible.com/ansible/dev_guide/developing_inventory.html)

**Comment by pgrange** [comment]
Thanks for the explanation. An I can see that you already explained that in documentation... shame on me.

Regards,
